File: algorithms/2160.js
var minimumSum = function(num) {
    num = String(num);
    const digits = num.split('');
    digits.sort((a,b) => Number(a) - Number(b));
    const first = Number(digits[0] + digits[2])   
    const second = Number(digits[1] + digits[3]);
    return first + second;
}

File: algorithms/2160.py
class Solution(object):
    def minimumSum(self, num):
        """
        :type num: int
        :rtype: int
        """
        digits = list(str(num))
        digits.sort()
        first = int(digits[0] + digits[2])
        second = int(digits[1] + digits[3])
        return first + second
        
